libathemecore/services.c

```diff
@@ -415,6 +415,8 @@ bool ircd_logout_or_kill(struct user *u, const char *login)
 	if (req.allowed)
```

Comment:
i'd probably prefer a set of curly braces on the other `if` here, to make things more obvious – it's not required though. @aaronmdjones thoughts on style here?